src/api/weather.js

```diff
@@ -12,6 +12,7 @@ async function getCityWeather(city) {
     const response = await instance.get(
       `data/2.5/weather?q=${city}&appid=${KEY}&units=metric`
     );
+    console.log(response.data)
     return response.data;
   } catch (error) {
     console.log("API Error", error);
```

Comment:
We need to remove try-catch mechanism to inform if the request is rejected, the file content should be:
```
import axios from "axios";

const KEY = "KEY";

const instance = axios.create({
  baseURL: "https://api.openweathermap.org/",
  timeout: 2000,
});

async function getCityWeather(city) {
  const response = await instance.get(
    `data/2.5/weather?q=${city}&appid=${KEY}&units=metric`
  );
  return response.data;
}

export default getCityWeather;

```